pipenv/patched/contoml/file/file.py

```diff
@@ -231,7 +231,7 @@ def has_anonymous_entry():
         if has_anonymous_entry():
             return items
         else:
-            return list(items) + [('', self[''])]
+            return items + [('', self[''])]
```

Comment:
Is this correct? I read the original issue description and it seems the old was correct; this would make the concat fail again because `items` is a `dict_items`, not list.

Reply:
Maybe a test is needed for this.

Reply:
That was a mistake when rebasing, revert it now. Thanks